### PR TITLE
Devise integration

### PR DESCRIPTION
**Pull request by Tramchandani**

### CODE REVIEW

File: db/migrate/20210401083517_devise_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.1]
```
Comment (kalpana19sharma) [inline db/migrate/20210401083517_devise_create_users.rb:3]: Why do you have 2 migrations? and what about the old users table?

**Review by kalpana19sharma — CHANGES_REQUESTED** [review]

File: db/migrate/20210401083517_devise_create_users.rb (hunk 1 above)
Comment (Tramchandani) [reply db/migrate/20210401083517_devise_create_users.rb:3]: Ma'am I had to delete the password digest and email column from the old table in order to add the email and password column of devise which was causing error. So I created new table using devise migration.

Comment (kalpana19sharma) [reply db/migrate/20210401083517_devise_create_users.rb:3]: Yeah but you have 2 files with `create_table :users` migration in this PR.

Comment (Tramchandani) [reply db/migrate/20210401083517_devise_create_users.rb:3]: Initially I was trying to integrate devise in the existing user table and late I create a new table hence the 2 tables. AddDevoceToUser migration is not used so I should delete it.

Comment (kalpana19sharma) [reply db/migrate/20210401083517_devise_create_users.rb:3]: yes

**Review by kalpana19sharma — APPROVED** [review]

**Pull request merged**